clients/src/main/resources/common/message/BrokerRegistrationRequest.json

```diff
@@ -51,7 +51,7 @@
     },
     { "name": "Rack", "type": "string", "versions": "0+", "nullableVersions": "0+",
       "about": "The rack which this broker is in." },
-    { "name": "IsMigratingZkBroker", "type": "bool", "versions": "1+", "default": "false",
-      "about": "Set by a ZK broker if the required configurations for ZK migration are present." }
+    { "name": "MigratingZkBrokerEpoch", "type": "int64", "versions": "1+", "default": "-1",
```

Comment:
can you add a comment to the top of `BrokerRegistrationRequest.json` stating that version 1 adds `MigratingZkBrokerEpoch`